packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/offer-details/index.js

```diff
@@ -0,0 +1,12 @@
+import angular from 'angular';
+
+import component from './component';
+
+const moduleName = 'ovhManagerDedicatedCloudBackupOfferDetails';
+
+angular
+  .module(moduleName, [])
```

Comment:
No translations, no ui kit, no routing ?

Reply:
Its more like a static component. So routing is not needed. It is used to display details of the offer inside the offer component.